auth-agents-common/src/main/java/org/apache/atlas/plugin/util/PolicyRefresher.java

```diff
@@ -211,17 +249,28 @@ public void run() {
 			DownloadTrigger trigger = null;
 			try {
 				trigger = policyDownloadQueue.take();
+				IS_RUNNING = true;
 
-				loadRoles();
-				loadPolicy();
-				loadUserStore();
+				if (trigger.roles) {
```

Comment:
will there be any issue if we reload only roles, and not policies ?

Reply:
There are a few actions where policies are not updated at all see this table - https://www.notion.so/atlanhq/ddd9cfebdbe349f995b98509dc63bd1f?v=ea7d68b59d9a435886bf9220d0bf3253&pvs=4